service/routes.go

```diff
@@ -5,17 +5,39 @@ import (
 	"github.com/rancher/go-rancher/api"
 	"github.com/rancher/go-rancher/client"
 	//This should be v2, supporting schemas
```

Comment:
@cjellick instead of using `github.com/rancher/go-rancher/v2` I'm using 1github.com/rancher/go-rancher/client1, because v2 wasn't supporting something, meaning there was an error which I can't recall right now but you had asked me to leave a comment somewhere so we can work on that later